openandroidinstaller/assets/configs/hero2lte.yaml

```diff
@@ -2,16 +2,21 @@ metadata:
   maintainer: Tobias Sterbak (tsterbak)
   device_name: Samsung Galaxy S7 Edge
   is_ab_device: false
-  device_code: hero2lte 
+  device_code: hero2lte
   supported_device_codes:
-    - hero2lte 
+    - hero2lte
```

Comment:
Are you sure these are all the same devices with the same hardware? Samsung is notorious for even having different cpus for different codes sometimes...

Reply:
Please check out the links of the respective commits.
The asserts basically mean that the lineage build can be flashed on all of these device codes.
If these models were diff socs, they wouldnt be in a single device tree.

Reply:
Ah great! Thank you! :)